CreeDictionary/res/crk.altlabel.tsv

```diff
@@ -19,86 +19,90 @@ FST TAG	LINGUISTIC (SHORT)	LINGUISTIC (LONG)	ENGLISH	NÊHIYAWÊWIN	EMOJI
 5Sg/Pl	5	Actor: 3rd Person Singular/Plural (Further Obviative)	s/he/they (further)	wiya/wiyawâw (nâha/nêki) -	
 5Sg/PlO	→ 5	Goal: 3rd Person Singular/Plural (Obviative)	→ him/her/them (further)	→ wiya/wiyawâw (nâha/nêki) -	
 					
-1Sg+2SgO			I → you (one)	niya → kiya	-
-1Sg+3SgO			I → him/her	niya → wiya (awa)	-
-1Sg+1PlO			I → we (not you)	niya → niyanân	-
-1Sg+12PlO			I → you and us	niya → kiyânaw	-
-1Sg+2PlO			I → you (all)	niya → kiyâwaw	-
-1Sg+3PlO			I → them	niya → wiyanaw (ôki)	-
-1Sg+4Sg/PlO			I → him/her/them	niya → wiya/wiyawâw (ana/aniki)	-
-1Sg+5Sg/PlO			I → him/her/them (further)	niya → wiya/wiyawâw (nâha/nêki)	-
+3Sg+I	3s	Actor: 3rd Person Singular (Inanimate)	it	ôma	
+3Pl+I	3s	Actor: 3rd Person Plural (Inanimate)	they	ôhi	
+4Sg+I	4s	Actor: 3rd Person Plural (Obviative) (Inanimate)	it (further)	kîkwaya	
```

Comment:
I especially had difficulty with this line!

Reply:
The convention that I've seen for existential actors that I've seen is `+0Sg`, and that could be extended to `+0Pl` and `+0SgO`, `+0PlO,` `+0Sg/Pl` and `+0Sg/PlO`. I'd have to think how that might apply to Cree morphology. Anyhow, that is what I use in the English phrase analysis, but I've been holding back due to potential consequences to the FST that I haven't wanted to resolve yet.

Reply:
But VII can both apply to existental actors (e.g., _kimiwan_) AND inanimate actors (_kinwâw_, e.g., _sîpîsis anima kinwâw_). So I don't thing `+0Sg` or `+0Pl` will work here :/

Reply:
I had used `+0Sg` etc. in the English phrase transcriptors, so because of that I've thought of using them hearing.

Currently, unless we divide VIIs into impersonals and those with inanimate subjects, for the `+0Sg` we'll have to provide the alternatives _it_ and _there is_, and for +0Pl the alternatives _they_ and _there are_. We can consider doing that split later on.